release note + data document added (N/N) (#192)
PediatricOpenTargets/OpenPBTA-analysis: doc/release-notes.md

# release notes

## current release
- release date: 2022-06-21
- status: available
- overview of changes:
  - This particular release added the following datasets:

>     - 11 RNA-Seq and WXS PNOC samples (plus 2 WXS PNOC results from previous release)

Reviewer: ```suggestion
    - 11 RNA-Seq and WXS PBTA samples (plus 2 WXS PBTA results from previous release)
```

# release notes

## current release
- release date: 2022-06-21
- status: available
- overview of changes:
  - This particular release added the following datasets:
    - 11 RNA-Seq and WXS PBTA samples (plus 2 WXS PBTA results from previous release)

>     - 1689 DGD targeted sequencing tumor samples

Reviewer: ```suggestion
    - 1801 DGD biospecimens
```

>     - 1689 DGD targeted sequencing tumor samples

Reviewer: ```suggestion
    - 1799 DGD tumor samples (929 DNA, 870 RNA)
```

    - methylation samples

>       - methylation samples: 

Reviewer: Is this supposed to say normal?

>       - methylation samples: 

Reviewer: Is this supposed to say normal?

>       - methylation samples: 

Reviewer: ```suggestion
      - methylation normal samples: 
```

>       - methylation samples: 

Reviewer: ```suggestion
      - methylation normal samples: 
```

        - 5 PBTA samples
        - 12 TARGET samples

>       - methylation sample type not reported
>         - 3 PBTA 

Reviewer: ```suggestion
```
I updated these (tumor)

>       - methylation sample type not reported
>         - 3 PBTA 

Reviewer: ```suggestion
```
I updated these (tumor)

      - methylation tumor samples

>         - 1143 PBTA samples

Reviewer: ```suggestion
        - 1146 PBTA samples
```

>         - 1143 PBTA samples

Reviewer: ```suggestion
        - 1146 PBTA samples
```

        - 751 TARGET samples

>       - RNA isoform merged TPM samples

Reviewer: ```suggestion
      - RSEM RNA isoform merged TPM samples
```

>       - RNA isoform merged TPM samples

Reviewer: ```suggestion
      - RSEM RNA isoform merged TPM samples
```

  - Update TCGA RNA-Seq data with GENCODE v36 GDC

>   - All additional changes as well as details about the above changes are as followed.

Reviewer: ```suggestion
  - All additional changes as well as details about the above changes are as follows.
```

>   - All additional changes as well as details about the above changes are as followed.

Reviewer: ```suggestion
  - All additional changes as well as details about the above changes are as follows.
```

- Add base histologies file to data release, see [ticket 333](https://github.com/PediatricOpenTargets/ticket-tracker/issues/333)

> - Update histologies file - key changes noted here (other tickets can be found in OpenPedCan repo)

Reviewer: ```suggestion
- Update histologies file - key changes:
```

> - Update histologies file - key changes noted here (other tickets can be found in OpenPedCan repo)

Reviewer: ```suggestion
- Update histologies file - key changes:
```

    - Change cancer_group of `Metastatic secondary tumors;Neuroblastoma` to `Neuroblastoma`
            - See discussion in [ticket 232](https://github.com/PediatricOpenTargets/ticket-tracker/issues/232)
            - [Ticket 234](https://github.com/PediatricOpenTargets/ticket-tracker/issues/234)
    - Update `primary_site` for some GMKF and TARGET NBL samples to resolve lumped together `primary_site` issues
            - [Ticket 257](https://github.com/PediatricOpenTargets/ticket-tracker/issues/257)
    - Update histologies file to use `pathology_diagnosis` (when available) or `cancer_group` (when path dx not available) to fill in `harmonized_diagnosis` 
            - [Ticket 259](https://github.com/PediatricOpenTargets/ticket-tracker/issues/259)

>      - DGD histologies cleanup: [ticket 325](https://github.com/PediatricOpenTargets/ticket-tracker/issues/325)

Reviewer: ```suggestion
```
We can remove this since it has never been released until now and was an upstream issue

>      - DGD histologies cleanup: [ticket 325](https://github.com/PediatricOpenTargets/ticket-tracker/issues/325)

Reviewer: ```suggestion
```
We can remove this since it has never been released until now and was an upstream issue

     - Short histology for PBTA cohort does not match: [ticket 309](https://github.com/PediatricOpenTargets/ticket-tracker/issues/309)

>      - Add extent of tumor regression: [ticket 298](https://github.com/PediatricOpenTargets/ticket-tracker/issues/298)

Reviewer: ```suggestion
     - Update extent of tumor resection: [ticket 298](https://github.com/PediatricOpenTargets/ticket-tracker/issues/298)
```

Oops, this was missed - updating now.
```suggestion
     - Add extent of tumor regression: [ticket 298](https://github.com/PediatricOpenTargets/ticket-tracker/issues/298)
```

>      - Add extent of tumor regression: [ticket 298](https://github.com/PediatricOpenTargets/ticket-tracker/issues/298)

Reviewer: ```suggestion
     - Update extent of tumor resection: [ticket 298](https://github.com/PediatricOpenTargets/ticket-tracker/issues/298)
```

Oops, this was missed - updating now.
```suggestion
     - Add extent of tumor regression: [ticket 298](https://github.com/PediatricOpenTargets/ticket-tracker/issues/298)
```

- Update EFO MONDO file 
    - Update EFO MONDO map file to contain new cancer group in v10 histologies file: [ticket 237](https://github.com/PediatricOpenTargets/ticket-tracker/issues/237)
    - Add NCIT column to the EFO MONDO map file + add new AML terms: [ticket 261](https://github.com/PediatricOpenTargets/ticket-tracker/issues/261)
- Update `ensg-hugo-pmtl-mapping.tsv` to remove NA line: [ticket 231](https://github.com/PediatricOpenTargets/ticket-tracker/issues/231)

- Re-run independent samples module since we would have cancer group update for the histologies file and addition of samples: [ticket 304](https://github.com/PediatricOpenTargets/ticket-tracker/issues/304)
- Add concatenated DGD MAF files to data release as `snv-dgd.maf.tsv.gz`: [ticket 248](https://github.com/PediatricOpenTargets/ticket-tracker/issues/248)

> - Add concatenated DGD fusion files as `fusion-dgd.tsv.gz`: [ticket 249](https://github.com/PediatricOpenTargets/ticket-tracker/issues/249)

Reviewer: move this up to earlier when showing DGD sample N (can just add ticket links and not need to describe

> - Add concatenated DGD fusion files as `fusion-dgd.tsv.gz`: [ticket 249](https://github.com/PediatricOpenTargets/ticket-tracker/issues/249)

Reviewer: move this up to earlier when showing DGD sample N (can just add ticket links and not need to describe

> - Update DNA and RNA delivery data files with the addition of PNOC samples: [ticket 253](https://github.com/PediatricOpenTargets/ticket-tracker/issues/253)

Reviewer: ```suggestion
- Update DNA and RNA delivery data files with the additional PBTA samples: [ticket 253](https://github.com/PediatricOpenTargets/ticket-tracker/issues/253)
```

> - Update DNA and RNA delivery data files with the addition of PNOC samples: [ticket 253](https://github.com/PediatricOpenTargets/ticket-tracker/issues/253)

Reviewer: ```suggestion
- Update DNA and RNA delivery data files with the additional PBTA samples: [ticket 253](https://github.com/PediatricOpenTargets/ticket-tracker/issues/253)
```

> - Re-run fusion filtering after PNOC samples added: [ticket 316](https://github.com/PediatricOpenTargets/ticket-tracker/issues/316)

Reviewer: ```suggestion
- Re-run fusion filtering: [ticket 316](https://github.com/PediatricOpenTargets/ticket-tracker/issues/316)
```
```suggestion
- Re-run fusion filtering after PNOC samples added: [ticket 316](https://github.com/PediatricOpenTargets/ticket-tracker/issues/316)
```

> - Re-run fusion filtering after PNOC samples added: [ticket 316](https://github.com/PediatricOpenTargets/ticket-tracker/issues/316)

Reviewer: ```suggestion
- Re-run fusion filtering: [ticket 316](https://github.com/PediatricOpenTargets/ticket-tracker/issues/316)
```
```suggestion
- Re-run fusion filtering after PNOC samples added: [ticket 316](https://github.com/PediatricOpenTargets/ticket-tracker/issues/316)
```

- Update focal CN preparation module to fine tune ENSEMBL, gene symbol and cytoband matching
   - [Ticket 320](https://github.com/PediatricOpenTargets/ticket-tracker/issues/320)
   - [Ticket 318](https://github.com/PediatricOpenTargets/ticket-tracker/issues/318)

> - After updating focal CN preparation module + addition of PNOC samples, modules need to re-run to update CNV annotation file: [ticket 254](https://github.com/PediatricOpenTargets/ticket-tracker/issues/254)

Reviewer: ```suggestion
- Rerun modules to generate CNV annotation file: [ticket 254](https://github.com/PediatricOpenTargets/ticket-tracker/issues/254)
```
```suggestion
- After updating focal CN preparation module + addition of PNOC samples, modules need to re-run to update CNV annotation file: [ticket 254](https://github.com/PediatricOpenTargets/ticket-tracker/issues/254)
```

> - After updating focal CN preparation module + addition of PNOC samples, modules need to re-run to update CNV annotation file: [ticket 254](https://github.com/PediatricOpenTargets/ticket-tracker/issues/254)

Reviewer: ```suggestion
- Rerun modules to generate CNV annotation file: [ticket 254](https://github.com/PediatricOpenTargets/ticket-tracker/issues/254)
```
```suggestion
- After updating focal CN preparation module + addition of PNOC samples, modules need to re-run to update CNV annotation file: [ticket 254](https://github.com/PediatricOpenTargets/ticket-tracker/issues/254)
```

- Run subtyping for all samples
    - [Ticket 299](https://github.com/PediatricOpenTargets/ticket-tracker/issues/299)
    - [Ticket 303](https://github.com/PediatricOpenTargets/ticket-tracker/issues/303)

- Update the download link to not only change the version from v10 to v11, but also change GENCODE versions: [ticket 270](https://github.com/PediatricOpenTargets/ticket-tracker/issues/270)
- Add bed files for TMB calculation and TMB results to data release 
    - [Ticket 250](https://github.com/PediatricOpenTargets/ticket-tracker/issues/250)
    - [Ticket 258](https://github.com/PediatricOpenTargets/ticket-tracker/issues/258)
- Add methylation array data for v11 release 
    - [Ticket 278](https://github.com/PediatricOpenTargets/ticket-tracker/issues/278)
    - [Ticket 269](https://github.com/PediatricOpenTargets/ticket-tracker/issues/269)
- Add TCGA with new GENCODE version to release
    - [Ticket 285](https://github.com/PediatricOpenTargets/ticket-tracker/issues/285)

>     - [Ticket 308](https://github.com/PediatricOpenTargets/ticket-tracker/issues/308)

Reviewer: methylation and TCGA are mentioned above, so I would move these up there

>     - [Ticket 308](https://github.com/PediatricOpenTargets/ticket-tracker/issues/308)

Reviewer: methylation and TCGA are mentioned above, so I would move these up there
